casper-private-auction-installer/src/main.rs

```diff
@@ -112,59 +123,75 @@ pub fn get_entry_points() -> EntryPoints {
 #[no_mangle]
 pub extern "C" fn call() {
     let entry_points = get_entry_points();
-    let auction_named_keys = data::create_auction_named_keys();
-    let auction_desig: String = runtime::get_named_arg("name");
+    let auction_named_keys = create_auction_named_keys();
+    let contract_name: String = runtime::get_named_arg(NAME);
     let (auction_hash, _) = storage::new_locked_contract(
         entry_points,
         Some(auction_named_keys),
-        Some(format!("{}_{}", auction_desig, data::AUCTION_CONTRACT_HASH)),
-        Some(format!("{}_{}", auction_desig, data::AUCTION_ACCESS_TOKEN)),
+        Some(format!("{contract_name}_{AUCTION_PACKAGE_HASH}")),
```

Comment:
We should investigate this, since this should be a contract package hash for consistency, but new_locked_contract returns only the contract hash

Reply:
Yes indeed I am also a bit circumspect about that one, it seems , the thing is that I got this habit from that page https://docs.casperlabs.io/dapp-dev-guide/tutorials/upgrade-contract/ where `contract_package_name` is used and actually the doc does not really say it's the package here.

```
pub fn new_locked_contract(entry_points: EntryPoints, named_keys: Option<NamedKeys>, hash_name: Option<String>, uref_name: Option<String>) -> (ContractHash, ContractVersion)
```

The old version was having

```
pub const AUCTION_CONTRACT_HASH: &str = "auction_contract_package_hash";
```

so it seems it's the package here and test don't pass if I change that.

Reply:
`hash_name: Option<String>` of the `new_contract` and `new_locked_contract` functions are the name of the `ContractPackageHash`, this inconsistency in the documentation was reported multiple times over the months.